ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -97,8 +97,16 @@ public protocol DrawerControllerDelegate: AnyObject {
 
     /// Called when drawer is being dismissed.
     @objc optional func drawerControllerWillDismiss(_ controller: DrawerController)
+    
     /// Called after drawer has been dismissed.
     @objc optional func drawerControllerDidDismiss(_ controller: DrawerController)
+    
+    /**
+     Called when drawer is getting dismissed when user tries to dismiss drawer by tapping in background, using resizing handle or dragging drawer to bottom.
+
+     Use this method to turn off drawer dismiss.
+     */
+    @objc optional func shouldDismissDrawer(_ controller: DrawerController) -> Bool
```

Comment:
so drawercontroller is popover in iPad regular screen size. what does this protocol work in those cases?

Reply:
Added support for same in popover too. Also updated PR description with demo video for popover.